docs/continuous_integration.rst

```diff
@@ -0,0 +1,60 @@
+.. _ci_tests:
+
+Continuous Integration
+===============================
+
+The SecureDrop project uses `GitHub Actions <https://docs.github.com/en/actions>`_ for
+running automated continuous integration on code changes. You can get an overview of what
+each project does by reviewing the ``Makefile`` and files in the ``.github/workflows`` folder.
+
+Basics
+------
+
+Our CI runs a mixture of linters, tests and build processes to validate code submissions.
+
+Most tasks have a corresponding ``make`` target that will run the same command locally. Common
+targets across all our projects include:
+
+* ``make lint``: run linters
+* ``make fix``: apply automated fixes from formatters and linters
+* ``make test``: run automated tests
+
+In CI, these are run in a container using the corresponding Linux distribution (e.g. Debian or Fedora),
+which can also be used to reproduce CI results locally. Some projects, like ``securedrop`` (server) and
+``securedrop-workstation``, automatically run commands in containers.
+
+Pull requests
+-------------
+
+Most CI jobs are triggered by both ``push`` and ``pull_request`` events. The former is run against
```

Comment:
Is it worth linking to the github docs for the [`push`](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#push) and [`pull_request`](https://github.com/freedomofpress/infrastructure/issues/4984#issuecomment-2305662370) events? For users not familiar with GHA those might be helpful references, but I'm not sure if it's necessarily worth including.

Reply:
Good idea, done.